src/views/List.jsx

```diff
@@ -1,22 +1,49 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [listFilter, setListFilter] = useState('');
+
+	const clickHandler = (event) => {
+		event.preventDefault();
+		setListFilter('');
+	};
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-			<ul>
-				{data.length > 0 ? (
-					<ul>
-						{data.map((item) => {
+			<form>
+				<label htmlFor="itemInput">Filter Items: </label>
+				<input
```

Comment:
If you press the Enter key while in the input, the form submits and reloads the page

Reply:
Nice catch @mentalcaries ! I have added a submit handler to the form that just prevents the default and that seems to have fixed the issue.